compilation_tests/ctest_constexpr_submdspan.cpp

```diff
@@ -71,7 +72,6 @@ dynamic_extent_1d() {
 
 MDSPAN_STATIC_TEST(dynamic_extent_1d<stdex::layout_left>());
 MDSPAN_STATIC_TEST(dynamic_extent_1d<stdex::layout_right>());
```

Comment:
these and the following tests won't work, since the constexpr_assert_equal can not be done as a constexpr expression for layout_stride anymore due to the dynamic storage construction. Its probably fixable but I couldn't figure it out ...